### ISSUE TITLE
Copy to Clipboard Behaviour

### ISSUE THREAD

**Issue by sbuethe** [olifolkerd/tabulator#1498]
Hi Oli,

I am currently using the copyToClipboard function to copy the table contents to the clipboard. I had to set "clipboard:true" to make this work. Unfortunately, with "clipboard:true" it is no longer possible to manually select text in the table with the cursor and copy it to the clipboard using ctrl+c or context menu. Whenever I do it, it copies a boolean "true" to the clipboard only.

Is there something I am missing here, is there a way around this or - if not - would it be possible to separate "copyToClipboard" from the clipboard event handling in a future update - therefore allowing to copy data to the clipboard in code (if needed) while still keeping the default (manual) browser behaviour?

Many thanks,

Sven

**Comment by olifolkerd** [comment]
Hey @sbuethe 

Thanks for getting in touch, that certainly dosnt sound right. im happy to look into that more for you over the weekend and push a fix if there is an issue.

Could i ask that when you raise issues in future that you use one of the templates you are offered as they will ask you the questions i most need answers to to fix the issue, including a copy of your table constructor so i can replicate your setup, and the version of Tabulator you are using and on what browser on what device.

Cheers

Oli :)

**Comment by sbuethe** [comment]
Sorry, you are right, I should have done this.

I updated to tabulator 4.04 with 1601 and the issue is still there. The browser I am using is Firefox 62.0.3 on Windows 10. Haven't done any testing on other browsers yet.

I investigated a bit further and as long as I only use the browser's context menu to copy selected text to the clipboard it works. As soon as I press ctrl + c  once, only "true" is copied to the clipboard from there on - no matter whether I use the context menu or the keyboard shortcut.

Latest testing happended with clipboard: 'copy' and I have not defined any accessors nor clipboardCopyFormatter or clipboardCopyStyled.
